apps/web/components/sections/project/media-player.tsx

```
@@ -0,0 +1,84 @@
+
+      <div className="thumbnails flex space-x-2">
+        {items.map((item, index) => (
+          <button
+            key={index}
+            onClick={() => setActiveIndex(index)}
+            className={`thumbnail w-16 h-16 border-2 rounded-md overflow-hidden flex items-center justify-center ${
+              activeIndex === index
+                ? "border-blue-500"
+                : "border-gray-300 hover:border-gray-400"
+            }`}
+          >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)